base/workflow/R/start_model_runs.R

```diff
@@ -231,30 +236,35 @@ start_model_runs <- function(settings, write = TRUE, stop.on.error = TRUE) {
             stdout_log = "launcher.out.log",
             stderr_log = "launcher.err.log",
             job_script = "launcher.sh",
-            qsub_extra = settings$host$modellauncher$qsub)
+            qsub_extra = settings$host$modellauncher$qsub.extra)
         }
         # HACK: Code below gets 'run' from names(jobids) so need an entry for
         # each run. But when using modellauncher all runs have the same jobid
         jobids[run] <- sub(settings$host$qsub.jobid, "\\1", out[length(out)])
       }
       
     } else {
+      pb <- utils::txtProgressBar(min = 0, max = nruns, style = 3)
+      pbi <- 0
+      for (run in job_modellauncher) {
         out <- PEcAn.remote::start_serial(
           run = run,
```

Comment:
With the loop missing this was set to the last run processed, which is likely to be a directory WITHOUT a launcher.sh in it.